cfme/tests/automate/test_automate_manual.py

```diff
@@ -1025,3 +1025,115 @@ def test_automate_task_schedule():
         1713072
     """
     pass
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1693362])
+def test_redhat_domain_sync_after_upgrade():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
```

Comment:
Please add caseimportance too

Reply:
@sshveta 
By default [case importance is high](https://github.com/ManageIQ/integration_tests/blob/master/conf/polarion_tools.yaml#L7) and I want these test cases with high importance. 
Hence I did not add the ```caseimportance``` tag.